internal/graphapi/permissions.go

```diff
@@ -0,0 +1,10 @@
+package graphapi
+
+const (
+	actionRootTenantCreate = "tenant_root_create"
```

Comment:
what's a root tenant? do we need special permissions for it?

Reply:
this has been removed. when permission checks are enabled, root tenants can't be created as there is no parent resource to check permissions for.

As discussed with the team, I implemented a local command to create root tenants.